plugins/blend/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = () => ({ addUtilities, theme, variants }) => {
+  addUtilities(
+    theme('blend').map(type => ({
+      [`.blend-${type}`]: {
+        'mix-blend-mode': type
+      }
```

Comment:
I'm wondering if we should take the same approach I took in a plugin for filters (which ill add in a new PR) where we configure blend once but also include the `background-blend-mode` utilities, e.g.
```js
      [`.blend-${type}`]: {
        'mix-blend-mode': type
      },
      [`.bg-blend-${type}`]: {
        'background-blend-mode': type
      },
```
Another thing to decide on would be the naming -- `bg-` is obvs a very crowded naming scheme already, but this feels the most predictable and unlikely to clash with anything

Reply:
Yeah good idea. I did run across a 3rd party plugin which combined both like this. I'll have to find it again and report back.

Reply:
Here it is. https://github.com/hacknug/tailwindcss-blend-mode I'll defer to you as to using this or extending ours.

Reply:
I would keep ours, there's no configuration here, which means intellisense is polluted and anyone who joins a project doesnt know the specific modes used in the project

Reply:
Ah good point about intellisense. 👍